Rakefile

```diff
@@ -1 +1,91 @@
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
+require 'rake/clean'
+require 'cucumber/rake/task'
+require 'mechanize'
+require 'fileutils'
+require 'yaml'
+require 'launchy'
+
+CDK_DOWNLOAD_URL='https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software'
+CDK_BOX_BASE_NAME='rhel-cdk-kubernetes-7.2-23.x86_64.vagrant'
+
+ADB_DOWNLOAD_URL='http://cloud.centos.org/centos/7/atomic/images'
+ADB_BOX_BASE_NAME='AtomicDeveloperBundle-2.0.0-CentOS7'
+
+CLOBBER.include('pkg')
+CLEAN.include('build')
+
+Cucumber::Rake::Task.new(:features)
+
+namespace :features do
+  cucumber_config = YAML.load_file('./.config/cucumber.yml')
+  cucumber_config.each_key do |key|
+    next if key == 'default'
+    Cucumber::Rake::Task.new(key.to_sym, "Run Cucumber features using profile '#{key}' (see cucumber.yml)'") do |t|
```

Comment:
Probably `see cucumber.yml` should be `see .config/cucumber.yml`. I had hard time to see `cucumber.yml` file then went to PR and found it is inside hidden folder `.config`.